docs/root/intro/arch_overview/advanced/matching/matching_api.rst

```diff
@@ -58,6 +58,17 @@ These input functions are available for matching TCP connections:
 * :ref:`Transport protocol <extension_envoy.matching.inputs.transport_protocol>`.
 * :ref:`Application protocol <extension_envoy.matching.inputs.application_protocol>`.
 
+.. _extension_category_envoy.matching.ssl.input:
+
+SSL Input Functions
+*******************
+
+These input functions are available for matching SSL connections:
```

Comment:
Where can these SSL inputs be used? Are they applicable for matching network streams and HTTP requests?

Reply:
>Where can these SSL inputs be used?

Currently, nowhere, but I want to use it in RBAC.

>Are they applicable for matching network streams and HTTP requests?

Yes. `Connection` has a `ssl` field, so authenticated inputs can be used for matching network streams.

In addition, I think it is necessary to separate network inputs in documentation. IP, port, SNI are available for listeners (`ConnectionSocket`) and network filters (`Connection`), but transport and application protocol are available only in listeners (`ConnectionSocket`). And the introduced authenticated inputs are available only in network filters (`Connection`).

Reply:
OK, we can fill the gaps with what inputs are available where later. IMHO we could apply URI SAN in filter chains, but we'd need some way to nest transport security and filter chains, which is not there yet.

Reply:
Could you mention that this works for both HTTP requests and TCP connections?